src/fromager/finders.py

```diff
@@ -90,29 +90,33 @@ def find_sdist(
 
 
 def find_wheel(
+    ctx: context.WorkContext,
     downloads_dir: pathlib.Path,
     req: Requirement,
     dist_version: str,
 ) -> pathlib.Path | None:
     filename_prefix = _dist_name_to_filename(req.name)
     canonical_name = canonicalize_name(req.name)
+    build_tag = ctx.settings.build_tag(req.name, dist_version)
```

Comment:
I wonder if the caller should pass this in, instead of us computing it inside? It feels a bit like the version. Maybe that means changing more places?

Reply:
I could do that, optionally pass `build_tag`